src/Dropdown/Dropdown.scss

```diff
@@ -1,11 +1,11 @@
-.dropdown.has-icon {
-  .icon-container {
-    max-width: 40px;
-    box-sizing: content-box;
+.dropdown {
+  margin-bottom: 1rem;
```

Comment:
Let's remove this. Consumers of the dropdown should decide if a margin is needed.

Reply:
Added [bootstrap spacing](https://getbootstrap.com/docs/4.3/utilities/spacing/) `mg-3` and updated icon to bootstrap spacing as well.